ember-bootstrap/addon/components/bs-modal/footer.hbs

```diff
@@ -1,15 +1,14 @@
-{{! @glint-nocheck }}
 {{#let (ensure-safe-component (bs-default @buttonComponent (component "bs-button"))) as |Button|}}
   <div class="modal-footer" ...attributes {{on "submit" (bs-default @onSubmit (bs-noop))}}>
     {{#if (has-block)}}
       {{yield}}
     {{else}}
       {{#if @submitTitle}}
         <Button @onClick={{@onClose}}>{{bs-default @closeTitle "Ok"}}</Button>
-        <Button @type={{bs-default @submitButtonType "primary"}} onClick={{@onSubmit}} disabled={{bs-default @submitDisabled false}}>{{@submitTitle}}</Button>
+        <Button @type={{bs-default @submitButtonType "primary"}} @onClick={{@onSubmit}} disabled={{bs-default @submitDisabled false}}>{{@submitTitle}}</Button>
```

Comment:
Good catch!

Is this causing a bug for consumers? If so, we may want to fix it in a separate PR and add missing test coverage.

Reply:
I've addressed this in #2123. The missing @ does not seem to be breaking anything, but it is nicer to have it there just to be sure.